core/src/components/toast/test/a11y/toast.e2e.ts

```diff
@@ -33,7 +49,11 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
       /**
        * IonToast overlays the entire screen, so
        * Axe will be unable to verify color contrast
-       * on elements under the toast.
+       * on elements under the toast. To avoid needing
+       * to spin up a controller toast using page.setContent
+       * (to support automatic dark theme testing), color
+       * contrast is checked on an inline toast in a
+       * separate test above.
        */
```

Comment:
Why does the color-contrast test work on the inline toast but not the controller toast?

Reply:
Something like this might work: https://github.com/ionic-team/ionic-framework/blob/fc88613fefa019a3b695a2c6e10c85cd3ce79ae8/core/src/components/toast/test/standalone/index.html#L16C1-L19

Reply:
Was able to get it working 👍